ratatui-core/src/text/text.rs

```diff
@@ -559,6 +561,25 @@ impl<'a> Text<'a> {
     }
 }
 
+impl UnicodeWidthStr for Text<'_> {
+    /// Returns the max width of all the lines.
```

Comment:
Is this supposed to be a note? I think we should document all of these functions or none of them 🤔

Reply:
Fair